packages/babel-plugin-transform-async-generator-functions/src/for-await.ts

```diff
@@ -1,31 +1,21 @@
 import { types as t, template } from "@babel/core";
 import type { NodePath } from "@babel/traverse";
 
-const buildForAwait = template(`
-  async function wrapper() {
-    var ITERATOR_ABRUPT_COMPLETION = false;
-    var ITERATOR_HAD_ERROR_KEY = false;
-    var ITERATOR_ERROR_KEY;
+const buildForAwait = template.statements(`
+  var ITERATOR_KEY = GET_ITERATOR(OBJECT), STEP_KEY = {}, NOT_DONE_KEY;
```

Comment:
Is there an upside to initializing STEP_KEY here? It'll be overwritten before use.

Reply:
Now it's no longer needed, thank you!